Main/ClientApp/src/Components/BaseCollectionView.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ReactNode } from "react";
+import { Collection } from "../Model/Collection";
+import ElementCreator from "./ElementCreator";
+import { GridList, GridListTile, Fab } from "@material-ui/core";
+import { Theme, createStyles, makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    list: {
+      padding: "25px 30px 25px 30px"
+    }
+  })
+);
+
+type BaseCollectionViewProps = {
+  collection: Collection | null | undefined;
+  children?: ReactNode;
+};
+
+export default function BaseCollectionView(props: BaseCollectionViewProps) {
+  const classes = useStyles();
+
+  return (
+    <>
+      <GridList
+        cols={3}
+        cellHeight="auto"
+        spacing={50}
+        className={classes.list}
+      >
+        {GridColumnList(0, props.collection)}
```

Comment:
just from looking at the code, I have no idea what it's supposed to do. Is the "modulo" meant to be a column number?